bridge/gitlab/import.go

```diff
@@ -185,10 +184,10 @@ func (gi *gitlabImporter) ensureNote(repo *cache.RepoCache, b *cache.BugCache, n
 			_, err = b.EditCommentRaw(
```

Comment:
`errResolve` here can never equal `cache.ErrNoMatchingOp` you think we should drop the first error check ?